src/modules/visualBuilder/pageBuilder.js

```diff
@@ -0,0 +1,146 @@
+export const initialiseCTBuilder = (url, variant, details) => {
+  document.addEventListener('DOMContentLoaded', () => onContentLoad(url, variant, details))
+}
+
+let container
+
+function onContentLoad (url, variant, details) {
+  document.body.innerHTML = ''
+  container = document.createElement('div')
+  container.id = 'overlayDiv'
+  container.style.position = 'relative' // Ensure relative positioning for absolute positioning of form
+  container.style.display = 'flex'
+  document.body.appendChild(container)
+  const overlayPath = 'https://d2r1yp2w7bby2u.cloudfront.net/js/lib-overlay/overlay.js'
+  loadOverlayScript(overlayPath, url, variant, details)
+    .then(() => {
+      console.log('Overlay script loaded successfully.')
+    })
+    .catch((error) => {
+      console.error('Error loading overlay script:', error)
+    })
+  loadCSS()
+  loadTypeKit()
+}
+
+function loadCSS () {
+  var link = document.createElement('link')
+  link.rel = 'stylesheet'
+  link.type = 'text/css'
+  link.href = 'https://d2r1yp2w7bby2u.cloudfront.net/js/lib-overlay/style.css'
+  document.head.appendChild(link)
+}
+
+function loadOverlayScript (overlayPath, url, variant, details) {
+  return new Promise((resolve, reject) => {
+    var script = document.createElement('script')
+    script.type = 'module'
+    script.src = overlayPath
+    script.onload = function () {
+      if (typeof window.Overlay === 'function') {
+        window.Overlay('#overlayDiv', url, variant, details)
+        resolve()
+      } else {
+        reject(new Error('ContentLayout not found in overlay.js'))
+      }
+    }
+    script.onerror = function (error) {
+      reject(error)
+    }
+    document.head.appendChild(script)
+  })
+}
+
+function loadTypeKit () {
+  var config = {
+    kitId: 'eqj6nom',
+    scriptTimeout: 3000,
+    async: true
+  }
+
+  var d = document
+  var h = d.documentElement
+  var t = setTimeout(function () {
+    h.className = h.className.replace(/\bwf-loading\b/g, '') + ' wf-inactive'
+    // $(document).trigger("TypeKitReady");
+  }, config.scriptTimeout)
+  var tk = d.createElement('script')
+  var f = false
+  var s = d.getElementsByTagName('script')[0]
+  var a
```

Comment:
meaningful name of the variables.
Also define them as `const`

Reply:
This is Done